lib/orbit/ocs/stream/pipeline.ex

```diff
@@ -96,4 +114,87 @@ defmodule Orbit.Ocs.Stream.Pipeline do
     # This is required per the framework, but could be useful down the line
     # to track which messages actually make it all the way into state.
   end
+
+  # Kinesis Stream Persistence
+
+  defp get_producer_opts(opts) do
+    resume_position = load_resume_position()
+    opts ++ [state: %{resume_position: resume_position}]
+  end
+
+  @spec load_resume_position() :: BroadwayKinesis.SubscribeToShard.starting_position()
+  defp load_resume_position do
+    cutoff = start_of_service_date()
```

Comment:
suggestion: could this instead be `midnight`?

Reply:
Renamed in 6c4e4e9